independent-publisher-connectors/Mensagia/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2330 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Mensagia",
+        "description": "Mensagia offer reliable SMS, Email and Voice for business automation communication.",
+        "version": "1.0",
+        "contact": {
+            "name": "Sistemes Informàtics ICON, S.L.",
+            "email": "asanteliz@iconsl.com",
+            "url": "https://www.iconsl.com/"
+        }
+    },
+    "x-ms-connector-metadata": [{
+            "propertyName": "Website",
```

Comment:
@AfriSR Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.